### ISSUE TITLE
Antibacterials

### ISSUE THREAD

**Issue by Jongmassey** [opensafely/codelist-development#133]
https://www.opencodelists.org/codelist/opensafely/antibacterials/27089eaa/

Migration of [openprescribing Psuedo-BNF format list](https://www.opencodelists.org/codelist/openprescribing/antibacterials/523e0707/) to dm+d format

**Comment by brianmackenna** [comment]
I think we should include more in the description. e.g. _This contains all x from y in the BNF_ 

@richiecroker - just looking at [the SQL for the measure](https://openprescribing.net/measure/ktt9_antibiotics/definition/), it list each individual code separately but is it all of section `0501` ?

**Comment by richiecroker** [comment]
@brianmackenna not quite, the measure code is [here](https://github.com/ebmdatalab/openprescribing/blob/main/openprescribing/measure_definitions/ktt9_cephalosporins.json)
It's missing:
5.1.4: Aminoglycosides
5.1.6: Clindamycin and lincomycin
5.1.7: Some other antibacterials
5.1.9: Antituberculosis drugs
5.1.10: Antileprotic drugs

Pretty sure this measure predates my time at the DataLab, but think the rationale was that the omitted sections are pretty specialist, and shouldn't be included when looking at percentages of "common" antibiotics.

**Comment by richiecroker** [comment]
Actually if we had time to refactor the measure, i'd probably prefer to explicitly exclude these in the measure rather than include everything else.

**Comment by Jongmassey** [comment]
If this is an opportunity to refactor I'm quite happy to make a start. If it's "common antibacterials" rather than all it should really be called that with an explanation of the rationale.

**Comment by richiecroker** [comment]
sorry, I looked at the wrong denominator.  This one _is_ [all of section 5.1 ](https://github.com/ebmdatalab/openprescribing/blob/main/openprescribing/measure_definitions/ktt9_antibiotics.json).  The point still stands for the quinolone etc marker though

**Comment by Jongmassey** [comment]
Queries have come from researchers at the [BRIT](https://britanalytics.uk/) project at Manchester University regarding inclusion criteria for antibiotics. Attached is a comparison of the two code lists - do any of the opensafely omissions seem worthy of consideration?

[antibiotic comparison.xlsx](https://github.com/opensafely/codelist-development/files/9350844/antibiotic.comparison.xlsx)

**Comment by Jongmassey** [comment]
Updated [BNF](https://www.opencodelists.org/codelist/openprescribing/antibacterials/663dc203/) 
based on [openprescribing measure definition](https://github.com/ebmdatalab/openprescribing/blob/510e8c07e841cd42284c109774d1730b6463f376/openprescribing/measure_definitions/ktt9_cephalosporins.json#L37-L46)
and subsequently exported [dm+d](https://www.opencodelists.org/codelist/opensafely/antibacterials/01fff52f/)

See also results in https://github.com/opensafely/codelist-development/issues/127 for impact on prescription counts in opensafely.

**Comment by Jongmassey** [comment]
@BillyZhongUOM please be aware that this is a replication of [an openprescribing measure](https://github.com/ebmdatalab/openprescribing/blob/510e8c07e841cd42284c109774d1730b6463f376/openprescribing/measure_definitions/ktt9_cephalosporins.json#L37-L46) and is not all of BNF 5.1. If you need a codelist with all of 5.1 I can show you how to do that on opencodelists.org

**Comment by BillyZhongUOM** [comment]
@Jongmassey  Hi Jon, Could you please show me how to do that on opencodelists.org? because I want to update our Brit overall_antibiotics codelist in the same way.
